### PR TITLE
Feature/107 issue credit note

### LINKED ISSUE #107: Admin - Issue a credit note to correct a wrong invoice
As an admin, if I generate an invoice that turns out to be wrong for any reason, I should **not be able to modify my invoice, as it is forbidden by law**.
Instead, I should be able to create a `credit_note`, which is a document that corrects the existing invoice.

### Rules
- I should not be able create a credit note if the subscription does not have an invoice
- If the subscription has an invoice, then I should be able to create as many credit notes as I want for that given subscription

### Tips
- Use the same workflow than for `signed_form` or `medical certificate`, but on the Admin side and using  the [`has_many_attached` ActiveStorage relationship](https://edgeguides.rubyonrails.org/active_storage_overview.html#has-many-attached)

### PR DESCRIPTION
**Pull request by silvadaniele**
This PR closes #107 
- Enables to generate a credit note;
- Enables to list and download all credit notes from the related subscription;
- Enables to identify the credit notes by it's id and creation date;

### CODE REVIEW

File: app/controllers/admin/credit_notes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Admin
+  class CreditNotesController < AdminController
+    before_action :set_subscription!, only: %i[show new create]
```
Comment (EmCousin) [inline app/controllers/admin/credit_notes_controller.rb:5]: Remove show

File: app/views/admin/subscriptions/show.html.erb [hunk 2]
```diff
@@ -27,6 +27,43 @@
...
+              <th>
+                <%= Subscription.human_attribute_name(:created_at) %> :
+              </th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <% @subscription.credit_notes.each do |credit_note| %>
+            <tr>
+              <th scope="row">
```
Comment (EmCousin) [inline app/views/admin/subscriptions/show.html.erb:54]: Don't use th inside a tbody tag. Only in a thead tag. Just use td

File: app/views/admin/subscriptions/show.html.erb [hunk 3]
```diff
@@ -27,6 +27,43 @@
...
       <% if @subscription.invoice.attached? %>
         <%= link_to t('defaults.download'), rails_blob_path(@subscription.invoice, disposition: "attachment") %>
+        <br>
+        <br>
+        <%= link_to t('.generate_credit_note'), new_admin_subscription_credit_note_path(@subscription.id), class: "btn btn-outline-success btn-sm" %>
+        <br>
+        <br>
+
+
+
```
Comment (EmCousin) [inline app/views/admin/subscriptions/show.html.erb:37]: Clear the wind

File: app/views/admin/subscriptions/show.html.erb [hunk 4]
```diff
@@ -27,6 +27,43 @@
       <strong><%= Subscription.human_attribute_name(:invoice) %> :</strong>
       <% if @subscription.invoice.attached? %>
         <%= link_to t('defaults.download'), rails_blob_path(@subscription.invoice, disposition: "attachment") %>
+        <br>
+        <br>
+        <%= link_to t('.generate_credit_note'), new_admin_subscription_credit_note_path(@subscription.id), class: "btn btn-outline-success btn-sm" %>
+        <br>
+        <br>
```
Comment (EmCousin) [inline app/views/admin/subscriptions/show.html.erb:34]: Don't abuse the br tags here. Use CSS or a bootstrap class to put some margin onto the link

File: config/locales/fr.yml [hunk 5]
```diff
@@ -153,6 +153,8 @@ fr:
         signed_form: Formulaire signé
         medical_certificate: Certificat médical
         invoice: Facture
+        credit_notes: Avoirs
+        created_at: Généré à
```
Comment (EmCousin) [inline config/locales/fr.yml:157]: Généré le

File: app/views/admin/subscriptions/show.html.erb (hunk 4 above)
Comment (silvadaniele) [reply app/views/admin/subscriptions/show.html.erb:34]: It was really ugly...sorry! =S

**Pull request merged**